### PR TITLE
Feat(discord): import single channel signals

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: https://github.com/status-im/status-desktop/issues/12238
Fixes: #12504 

### What does the PR do

Introducing import single discord channel functionality  

### Affected areas

Communities

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

When the import was successfully completed:
Снимок экрана 2023-11-03 в 16 10 58

When the import was successfully completed with warnings:
Снимок экрана 2023-11-03 в 16 12 00

When the user tried to import already imported channel:
Снимок экрана 2023-11-03 в 16 24 43

Imported channel:
Снимок экрана 2023-11-03 в 16 12 30

### CODE REVIEW

**Comment by borismelnik** [comment]
https://github.com/status-im/status-desktop/pull/12458 - the history of this feature there

File: ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml [hunk 1]
```diff
@@ -236,7 +237,9 @@ QtObject {
                                              // TODO
                                          }
                                       }, from = 0) {
-        console.warn("!!! IMPLEMENT ME requestImportDiscordChannel") // FIXME
+        communitiesModuleInst.requestImportDiscordChannel(args.name, args.discordChannelId, args.communityId,
+                                                        args.description, args.color, args.emoji, from)
+//        console.warn("!!! IMPLEMENT ME requestImportDiscordChannel") // FIXME
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml:242]: You can remove this now :)

**Review by caybro — APPROVED** [review]
Awesome, looks nice!

**Review by jrainville — APPROVED** [review]
LGTM!

**Pull request merged**